src/utils/leaderboardUtils.ts

```
@@ -0,0 +1,74 @@
+  currentGig,
+  setlist
+}: {
+  player: GameState['player']
+  lastGigStats: GigStats | null
+  currentGig: GigData | null
+  setlist: GameState['setlist']
+}) => {
+  if (!player.playerId || !player.playerName) return
+
+  // Create a unified list of song stats to submit
+  let songsToSubmit
```

Biome (2.4.11)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)